src/actions.js

```diff
@@ -6,7 +6,11 @@ const actionCreator = type => payload => ({
 export const OPTIONS_LOADED_TYPE = 'REDUXCRUD_OPTIONS_LOADED';
 export const OPTIONS_LOAD_FAILED_TYPE = 'REDUXCRUD_OPTIONS_LOAD_FAILED';
 export const URL_MATCH_FAILURE_TYPE = 'REDUXCRUD_URL_MATCH_FAILURE';
+export const FORM_SUBMISSION_SUCCEEDED_TYPE = 'REDUXCRUD_FORM_SUBMISSION_SUCCEDEED_TYPE';
+export const FORM_SUBMISSION_FAILED_TYPE = 'REDUXCRUD_FORM_SUBMISSION_FAILED_TYPE';
 
 export const OPTIONS_LOADED = actionCreator(OPTIONS_LOADED_TYPE);
 export const OPTIONS_LOAD_FAILED = actionCreator(OPTIONS_LOAD_FAILED_TYPE);
 export const URL_MATCH_FAILURE = actionCreator(URL_MATCH_FAILURE_TYPE);
+export const FORM_SUBMISSION_SUCCEEDED = actionCreator(FORM_SUBMISSION_SUCCEEDED_TYPE);
```

Comment:
in Redux event stack we will get sometgin like that ?
FORM_SUBMISSION_SUCCEEDED.REQUEST ?
if yes it will look little strange